sys/netinet/ip6.h

```diff
@@ -97,10 +97,12 @@ struct ip6_hdr {
 #if BYTE_ORDER == BIG_ENDIAN
 #define IPV6_FLOWINFO_MASK	0x0fffffff	/* flow info (28 bits) */
 #define IPV6_FLOWLABEL_MASK	0x000fffff	/* flow label (20 bits) */
+#define IPV6_FLOW_ECN_MASK	0x00300000	/* the 2 ECN bits */
```

Comment:
I wonder if we should go with the "FLOW" and name it `IPV6_FLOWECN_MASK` instead?

Reply:
I suspect that compatibility is a bit more useful than consistency here.

Reply:
... which is to say that I think it's ok as-is. :)